addons/crm_iap_lead_enrich/data/mail_template.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="lead_enrichment_no_credits" model="mail.template">
```

Comment:
This template is quite dummy and does not really work as it is defined. I would instead make a template on res.users as you are trying to contact users, with partner_to correctly set.

Reply:
As I'm am not contacting users anymore but the admin in charge of the IAP service, I think I should still use iap.model_iap_account. But I now set <field name="email_to">${object.create_uid.email}</field> directly in the template.
and not in the values passed to the template.